blockchain/fullblocktests/generate.go

```diff
@@ -2142,7 +2142,7 @@ func Generate(includeLargeReorg bool) (tests [][]TestInstance, err error) {
 	g.SetTip("brs3")
 	g.NextBlock("bmf35", outs[15], ticketOuts[15], func(b *wire.MsgBlock) {
 		spend := chaingen.MakeSpendableStakeOut(b, 5, 2)
-		ticketPrice := dcrutil.Amount(b.STransactions[5].TxOut[0].Value)
+		ticketPrice := dcrutil.Amount(g.CalcNextRequiredStakeDifficulty())
```

Comment:
This is not correct.  It will calculate the ticket price for the *next* block, not the current one.